doc/graph/fusion_patterns/sdpa.md

```diff
@@ -128,9 +128,9 @@ platforms follow the general description in @ref dev_guide_data_types.
 4. GPU
    - Optimized implementation is available for 4D Q/K/V tensors with shape
      defined as (N, H, S, D).
-   - Optimized implementation is available for floating-point SDPA with `f16`
-     data type and `D <= 256` on Intel Graphics Products with Intel(R) Xe Matrix
-     Extensions (Intel(R) XMX) support.
+   - Optimized implementation is available for `f16` or `bf16` SDPA with `f32`
+     intermediate data type and `D <= 256` on Intel Graphics Products with
```

Comment:
What does intermediate data type mean when users construct graph? Do we need to describe it in text or picture?

Reply:
I'm not sure how to make it better. Please feel free to make a suggestion here. Thanks!

Reply:
From the picture in the description, the intermediate data type means the data type starting from the output of first MatMul, to the input of Softmax. Maybe we can add an intermediate dtype mark on the picture.
![image](https://github.com/user-attachments/assets/dbc3c6bd-95fb-4c6b-88c6-2d5996a1cf67)

And then describe the relationship between input and intermediate dtypes.